app/models/concerns/smooch_search.rb

```diff
@@ -161,7 +161,7 @@ def search_for_similar_published_fact_checks(type, query, team_ids, limit, after
 
     # "type" is text, video, audio or image
     # "query" is either a piece of text of a media URL
-    def search_for_similar_published_fact_checks_no_cache(type, query, team_ids, limit, after = nil, feed_id = nil, language = nil)
+    def search_for_similar_published_fact_checks_no_cache(type, query, team_ids, limit, after = nil, feed_id = nil, language = nil, published_only = true)
```

Comment:
I believe the request in the ticket was to avoid using the `_no_cache` version of the search so that we would be able to take advantage of the performance speedup from caching?

Reply:
@skyemeedan I implemented the cache in project_media.rb file as this method called by tipline and relevant articles do I cache the relevant articles result only

Reply:
My understanding is there are two kinds of caching in discussion:
1) cache the results served to the frontend so we can later save them (you implemented thids)
2) Alegre similarity caching (if we've already recently served a request for a piece of media, and we get a second one, we can just respond with the cache instead of doing the lookup) @DGaffney is this right?

I think the intention was to also change the "search_for_similar_published_fact_checks" function to use the cached version so that it can return more quickly.